CONTRIBUTING.md

```diff
@@ -92,6 +92,7 @@ npm run fix # Runs both of the above commands in parallel
 npm run test:watch # Runs karma on Chrome, re-running when source files change
 
 npm test # Lints all files, runs karma, and then runs coverage enforcement checks.
+npm run test:closure # Runs closure build tests against all closurized files
```

Comment:
Fair enough! I can make it part of `posttest` (which automatically gets run when `npm test` is run) and then modify our CI matrix so that it runs something like `npm run test:unit` for our unit tests when used for that particular build matrix.